### ISSUE TITLE
Phonetic fonts are incorrectly displayed

### ISSUE THREAD

**Issue by nguyenlean96** [goldendict/goldendict#1347]
This issue happens for all the resources, every single word.

![penalise_ifd](https://user-images.githubusercontent.com/65287357/106764487-d20eff80-6605-11eb-94fe-0a2a4e6861f4.PNG)
![syntax_ifd](https://user-images.githubusercontent.com/65287357/106764489-d2a79600-6605-11eb-9777-130e673dc237.PNG)
![aficinados_ifd](https://user-images.githubusercontent.com/65287357/106764491-d2a79600-6605-11eb-8adf-91b13916b1ab.PNG)

I though they were caused by "Lucida Sans Unicode" font but after downloading and installing a stock one, there were nothing changed 😭 It used to work fine before I changed my computer @@

Please help me on this one, please!!!

**Comment by nikita-moor** [comment]
Yes, it looks as if the font is missing. It's a problem of the dictionary, not GoldenDict. You could try finding what font is assigned to these phonetic transcriptions, but you should be somewhat experienced in HTML:

- right-click on the word
- choose "Inspect"
